gateway/1.8/modules/ROOT/pages/_partials/task-config-readiness-liveness.adoc

```diff
@@ -70,7 +70,7 @@ To modify the default parameters, update your Helm chart. You can modify these p
 NOTE: When using a readiness probe in Kubernetes (or Openshift) and Connected Mode, omit the `--wait` flag in the Helm chart used to deploy Flex Gateway. Not doing so might result in an installation failure because Helm waits for a readiness state of `true`. If there's no API deployed to your Flex Gateway instance, the readiness state is `false`. This is not an issue with custom readiness probe configurations or gateways that have an API configured.
 +
 [discrete]
-==== Configure External Components
+=== Configure External Components
```

Comment:
I'd included four "=" characters so that the heading text wouldn't appear so large within the tab. It was a cosmetic choice, which IMO looks better. If you want to make it a "===" then I think you can remove the "[discrete]", as that just allows for improperly structured headings. But make sure to build locally to check it all out.